src/types.ts

```diff
@@ -3,6 +3,12 @@ export type menuProps = {
   path: string;
 };
 
+export enum Sort {
+  ALL = "all",
+  RATING = "rating",
+  ORDER = "order",
+}
```

Comment:
it is a good idea! I learned how to use it through fem frågor uppgiften! It looks neat and gooood